docs/azure/includes/dotnet-all.md

```diff
@@ -202,7 +202,7 @@
 | Image Search | NuGet [2.0.0](https://www.nuget.org/packages/Microsoft.Azure.CognitiveServices.Search.ImageSearch/2.0.0) |  |  |
 | Information Protection | NuGet [1.7.147](https://www.nuget.org/packages/Microsoft.InformationProtection.File/1.7.147) |  |  |
 | Insights | NuGet [0.16.0-preview](https://www.nuget.org/packages/Microsoft.Azure.Insights/0.16.0-preview) |  |  |
-| IoT Device Update | NuGet [1.0.0-beta.2](https://www.nuget.org/packages/Azure.IoT.DeviceUpdate/1.0.0-beta.2) | [docs](/dotnet/api/overview/azure/IoT.DeviceUpdate-readme-pre/) | GitHub [1.0.0-beta.2](https://github.com/Azure/azure-sdk-for-net/tree/Azure.IoT.DeviceUpdate_item.Version/sdk/deviceupdate/Azure.Iot.DeviceUpdate/) |
```

Comment:
@weshaggard It looks like we're getting bad paths in the file that's getting delivered to us. Is there something that can be done upstream?

cc: @scottaddie @gewarren

Reply:
@CamSoper thanks for pointing this out. I will get the automation fixed for this.